<?xml version="1.0" ?>

<launch>
  <param name="robot_description" command="$(find xacro)/xacro --inorder $(find franka_description)/robots/franka_emika_01_01.urdf.xacro" />
  <node name="franka_joint_state_publisher" pkg="franka_visualization" type="franka_joint_state_publisher" output="screen">
    <rosparam command="load" file="$(find franka_visualization)/param/robot_read_settings.yaml" />
  </node>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_description)/rviz/franka_description.rviz"/>
</launch>
